app/src/main/res/values-fr/strings.xml

```diff
@@ -1,78 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="action_settings">Réglages</string>
-    <string name="name_prompt">Quel devrait être ton nom?</string>
-    <string name="scan_invitation">Scannez ce QR-Code</string>
+    <string name="action_settings">Paramètres</string>
+    <string name="name_prompt">Quel devrait être votre nom \?</string>
+    <string name="scan_invitation">Scannez ce code QR</string>
     <string name="scan_invited">Numérisation de code QR</string>
     <string name="paste_invitation">Coller les données ici</string>
     <string name="invalid_data">Données invalides</string>
-    <string name="invalid_name">Nom Incorrect</string>
+    <string name="invalid_name">Nom incorrect</string>
     <string name="cancel">annuler</string>
     <string name="next">suivante</string>
     <string name="enter_password">Entrer le mot de passe</string>
     <string name="password">Mot de passe</string>
     <string name="camera_permission_request">La permission de la caméra est nécessaire pour scanner les codes QR</string>
-    <string name="invalid_qr">QR-Code invalide</string>
+    <string name="invalid_qr">Code QR invalide</string>
     <string name="network_connect_invitation">Aucune information disponible, veuillez vous connecter à un réseau</string>
-    <string name="contact_has_no_address_warning">Attention: le contact n'a pas d'adresse</string>
-    <string name="menu_settings">Réglages</string>
+    <string name="contact_has_no_address_warning">Attention : le contact n’a pas d’adresse</string>
+    <string name="menu_settings">Paramètres</string>
     <string name="menu_license">Licence</string>
     <string name="menu_backup">Sauvegarde</string>
     <string name="menu_about">À propos</string>
-    <string name="call_connecting">Appel...</string>
+    <string name="call_connecting">Appel …</string>
     <string name="settings_change_name">Changer de nom</string>
     <string name="settings_change_language">Changer de langue</string>
-    <string name="settings_change_address">Changement d'adresse</string>
+    <string name="settings_change_address">Changement d’adresse</string>
     <string name="settings_change_password">Changer le mot de passe</string>
     <string name="settings_nightmode">Mode nuit</string>
     <string name="settings_ignore_unsaved">Ignorer les appelants inconnus</string>
-    <string name="about_source">Code source:</string>
-    <string name="about_author">Développeurs:</string>
-    <string name="about_license">Licence:</string>
+    <string name="about_source">Code source :</string>
+    <string name="about_author">Développeurs :</string>
```

Comment:
I was about to ask you if that space is a french thing. But a friend next to be pulled out a french book and assured me. XD